### DISCUSSION TITLE
Overlay

### DISCUSSION THREAD

**Discussion by nebunebu** [maltejur/discord-screenaudio#179] [Q&A]
Is there an overlay that works with this? How would you get discover-overlay to work with this?

**Comment by maltejur** [comment]
Not really, to quote the readme:

> The purpose of this project is not to provide an alternative to the original Discord client. Rather, it should be used in addition to the original client in case you want to stream something, maybe used with a second account. For anything else, this client has way too many things that work less well than in the original client.

But if you use KDE and want to try something hacky, there is this effect called "Thumbnail Aside", which will display a small version of any window in the bottom right corner. You could use that as a kind of overlay.

![Screenshot_20230917_223108](https://github.com/maltejur/discord-screenaudio/assets/48161361/68816add-3d6c-4e57-8424-6a43e84110ac)
